### PR TITLE
Update destination-event-streaming-overview.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destination-event-streaming-overview.md [hunk 1]
```diff
@@ -3,33 +3,42 @@ title: Event Streaming Overview
 description: Send Amplitude data to other tools in your stack with just a few clicks, using no-code event streaming integrations. 
 ---
 
-Event streaming lets you send the data in Amplitude across your stack. With event streaming, you can use the rich behavioral data in Amplitude to enrich customer profiles and stream data to marketing, sales, and infrastructure tools.
+Event Streaming empowers you to share your Amplitude data seamlessly throughout your entire system. It allows you to utilize the valuable behavioral data in Amplitude to enhance customer profiles and send data to your marketing, sales, and infrastructure tools.
```
Comment (markzegarelli) [inline docs/data/destination-event-streaming-overview.md:6]:
```suggestion
Event Streaming lets you share your Amplitude data throughout your entire system. Use the valuable behavioral data in Amplitude to enhance customer profiles and send data to your marketing, sales, and infrastructure tools.
```

@Brandoncharleskhoo1 What "sytstem" are you talking about here?

File: docs/data/destination-event-streaming-overview.md [hunk 2]
```diff
@@ -3,33 +3,42 @@ title: Event Streaming Overview
...
 
-Event streaming includes powerful, no-code, configuration-based tools that give fine-grained control over the data you send. Filter by user, group, and event properties to control what you forward to your other tools. You can also see important metrics like event volume, end-to-end latency, and detailed Delivery Issue information to understand the performance and health of your integration. 
+With event streaming, you gain access to user-friendly, configuration-based tools that offer precise control over the data you transmit. You can filter data by user, group, and event properties, ensuring that only the relevant information is delivered to your downstream tools. Additionally, you can monitor key metrics such as event volume, latency, and detailed delivery status to assess the performance and reliability of your streaming integration.
```
Comment (markzegarelli) [inline docs/data/destination-event-streaming-overview.md:8]:
```suggestion
With event streaming, you gain access to user-friendly, configuration-based tools that offer precise control over the data you send. Filter data by user, group, and event properties, ensuring that you send only the relevant information to your downstream tools. Additionally, you can monitor key metrics like event volume, latency, and detailed delivery status to assess the performance and reliability of your streaming integration.
```

File: docs/data/destination-event-streaming-overview.md [hunk 3]
```diff
@@ -3,33 +3,42 @@ title: Event Streaming Overview
...
-- **Starting from Event Streaming Setup:** Amplitude's streaming integrations focus on data from the setup point forward. Historical data isn't included in this process, which ensures that Amplitude transmits only events captured post-configuration.
-- **Reliable Retry Mechanism:** Amplitude addresses intermittent errors through in-memory retries with exponential backoff for initial sends. A robust retry pipeline attempts up to 10 times within a 4-hour window to handle retry errors. Amplitude applies this mechanism to all Event Streaming destinations.
-- **Streamlined Monitoring and Management:** The Event Streaming Debugger UI in Amplitude Data allows users to monitor pending retries and their progress. Expired payloads that exhaust retry attempts are clearly marked. The UI provides insights into error categories and also offers samples of failed payloads for analysis.
+- **Billing efficiency:** Amplitude tracks event volume based on distinct events sent. If the same event is sent to multiple Event Streaming destinations, it's counted only once for billing.
```
Comment (markzegarelli) [inline docs/data/destination-event-streaming-overview.md:12]:
```suggestion
- **Billing efficiency:** Amplitude tracks event volume based on distinct events sent. If you send same event to multiple Event Streaming destinations, it's counted only once for billing.
```

File: docs/data/destination-event-streaming-overview.md [hunk 4]
```diff
@@ -3,33 +3,42 @@ title: Event Streaming Overview
...
+- **Latency target:** Amplitude aims for an end-to-end p95 latency of 60 seconds, monitored and supported by alerts.
+- **Retry mechanism:** Intermittent errors are addressed through in-memory retries with exponential backoff for initial sends. A robust retry pipeline attempts up to 10 times within a 4-hour window. This mechanism applies to all Event Streaming destinations.
```
Comment (markzegarelli) [inline docs/data/destination-event-streaming-overview.md:14]:
```suggestion
- **Retry mechanism:** Amplitude addresses Intermittent errors using in-memory retries with exponential backoff for initial sends. The retry pipeline attempts up to 10 times within a 4-hour window. This mechanism applies to all Event Streaming destinations.
```

File: docs/data/destination-event-streaming-overview.md [hunk 5]
```diff
@@ -3,33 +3,42 @@ title: Event Streaming Overview
...
+- **Streamlined monitoring and management:** The Event Streaming Debugger UI in Amplitude Data enables users to monitor pending retries, progress, and expired payloads. Insights into error categories and failed payload samples are provided for analysis.
```
Comment (markzegarelli) [inline docs/data/destination-event-streaming-overview.md:15]:
```suggestion
- **Streamlined monitoring and management:** The Event Streaming Debugger UI in Amplitude Data lets you monitor pending retries, progress, and expired payloads. Analyze failed payload samples to gain insight into error categories.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by Brandoncharleskhoo1** [comment]
Closing becauuse it has a conflict and I don't know how to resolve it. Created a new PR https://github.com/amplitude/amplitude-dev-center/pull/186

**Pull request closed without merging**